docs/stories/TF-1786-TeamMail-Swipe-in-Email-item-in-ThreadView .md

```diff
@@ -0,0 +1,97 @@
+# Summary
+
+* [Related EPIC](#related-epic)
+* [Definition](#definition)
+* [Screenshots](#screenshots)
+* [Misc](#misc)
+
+## Related EPIC
+
+* TODO: {link}
+
+## Acceptance Criteria: 
+```
+GIVEN I'm a Tmail user
+AND There are existed uread emails on my mailbox
+WHEN I swipe left to right on an email rows
+THEN I see the email row is swiped as blue background color belong with "read email icon"
+AND System mark an email as read
+
+GIVEN I'm a Tmail user
+AND There are existed read emails on my mailbox
+WHEN I swipe left to right on an email rows
+THEN I see the email row is swiped as blue background color belong with "unread email icon"
+AND System mark an email as unread
```

Comment:
How often do you unread a mail?

Me not very often!

I suggest choosing another action for swipe right. Eg Move the mail : open the dialog to move an email.

Reply:
IMO swipe action always one-step action. Move is complicated.

How about 'mark star'?

Reply:
Ma be delete action? after you read it, you continue to swipe to move to Trash to continue swiping next email but not need to move you finger (I think archive is appropriate but we do  not have it now )